pkg/config/parameters.go

```diff
@@ -802,6 +802,7 @@ func (a AccessLogLevel) Validate() error {
 
 const LogLevelInfo AccessLogLevel = "info" // Default log level.
 const LogLevelError AccessLogLevel = "error"
+const LogLevelServerError AccessLogLevel = "server_error"
```

Comment:
Is there discrepancy here, since the log level name used elsewhere in the PR is `critical`?

Reply:
fixed in e639dc180ee3f7ad9ddf4adbafce77b85f0996d9